README.md

```diff
@@ -310,6 +310,24 @@ package:
     # same as -s ; SYFT_PACKAGE_CATALOGER_SCOPE env var
     scope: "squashed"
 
+  # enable specific language or ecosystem cataloger
+  # default: all catalogers are enabled by default
```

Comment:
nit: mentioning that all catalogers are enabled by default is a little misleading. There is still a selection of which catalogers to use based off of the source (dir scan or image scan)